less/items.less

```
@@ -138,6 +138,14 @@
         .description-edit {
           margin-inline-start: auto;
           opacity: 20%;
+
+          .fa-solid.fa-edit {
+            position: absolute;
+            top: 0;
+            right: 0;
+            bottom: 0;
```

I've been trying to switch to using more logical properties where possible, so these should be `inset`:

```suggestion
            inset-block: 0;
            inset-inline-end: 0;
```